store/CSDK/main.cpp

```diff
@@ -21,6 +21,7 @@
 #include <iostream>
```

Comment:
Can we move this to test ? because this is not a product code.

Reply:
we just have to create a folder and move this file to that folder. other files we keep in source folder.
update cmakelist with proper path. 
This gives good idea for new developer to identify which is source and which is test code.

Reply:
OK，done